test/org/zalando/stups/mint/worker/job/sync_app_test.clj

```diff
@@ -60,7 +61,7 @@
         kio-app {:id "kio"
                  :active false}]
     (with-redefs [storage/get-app (constantly test-app-details)
-                  s3/writable? (constantly true)
+                  s3/put-string (constantly nil)
```

Comment:
It also tests that the defmulti `writable?` is called correctly inside `sync_app.clj`.